R/grid_code_paths.R

```diff
@@ -225,6 +225,8 @@ tune_grid_loop_impl <- function(fn_tune_grid_loop_iter,
 
     seeds <- generate_seeds(rng, n_splits * n_grid_info)
 
+    slice_seeds <- slice_seeds
```

Comment:
Yeah, this isn't great, but it's probably what needs to happen now. Perhaps @HenrikBengtsson has a suggestion for this.

Reply:
The other option here is [`.options.future = list(globals = structure(..., add = ...))`](https://github.com/HenrikBengtsson/doFuture/blob/95b8e80db6eeb4017eb6be645fe54c817b7dcbb7/R/dofuture_OP.R#L26-L34).

Reply:
> The other option here is [`.options.future = list(globals = structure(..., add = ...))`](https://github.com/HenrikBengtsson/doFuture/blob/95b8e80db6eeb4017eb6be645fe54c817b7dcbb7/R/dofuture_OP.R#L26-L34).

Another way is to "guide" Futureverse (under the hood it's the **globals** package) by adding a dummy use of the global variable that it fails to detect, e.g.

```r
f <- future({ 
  b # "declare" 'b' as a global variable
  a * get("b") 
})
```

which effectively works the same as:

```r
f <- future({ 
  a * get("b") 
}, globals = structure(TRUE, add = "b"))
```

This is mentioned in https://future.futureverse.org/articles/future-4-issues.html#missing-globals-false-negatives